qutebrowser/html/settings.html

```diff
@@ -13,22 +13,112 @@
 {% endblock %}
 
 {% block style %}
-table { border: 1px solid grey; border-collapse: collapse; }
-pre { margin: 2px; }
-th, td { border: 1px solid grey; padding: 0px 5px; }
-th { background: lightgrey; }
-th pre { color: grey; text-align: left; }
-input { width: 98%; }
-.setting { width: 75%; }
-.value { width: 25%; text-align: center; }
-.noscript, .noscript-text { color:red; }
-.noscript-text { margin-bottom: 5cm; }
-.option_description { margin: .5ex 0; color: grey; font-size: 80%; font-style: italic; white-space: pre-line; }
+table {
+    border-spacing: 10px;
+}
+
+tbody tr:nth-child(odd) {
+    background: #eaf4fb;
+}
+
+pre {
+    margin: 2px;
+}
+
+th {
+    padding: 10px;
+    border-radius: 5px;
+    background: #a6dfff;
+    text-align: left;
+    font-weight: normal;
+    font-size: 1.5rem;
+    color: #084c88;
+}
+
+td {
+    padding: 5px 5px;
+}
+
+th pre {
+    color: grey;
+    text-align: left;
+}
+
+input {
+    padding: 8px;
+    width: 98%;
+    box-sizing: border-box;
+    border-radius: 4px;
+    border: 1px solid #01cdd0;
+    font-size: 0.9rem;
+    font-family: DejaVu, serif;
+}
+
+input:focus {
+    outline: none;
+    border: 2px solid #7a589ea6;
+}
+
+/* Hides the radio button */
+input[type="radio"] {
+    position: absolute;
+    opacity: 1%;
+    width: min-content;
+    margin: 0;
+}
```

Comment:
I'm a bit confused on how this works - wouldn't something like `display: none` suffice?

FWIW this also doesn't seem to work properly with an older QtWebEngine version - you should be able to test with `tox -e py39-pyqt512 --notest` (adjust `py39` if you're using a different Python version), followed by `.tox/py39-pyqt512/bin/python3 -m qutebrowser --temp-basedir :set`:

![image](https://user-images.githubusercontent.com/625793/144001070-d2d67da8-6cd2-41d5-ae2a-a4516157d3d5.png)

(looks like `opacity:` in CSS [was added with Chrome 78](https://caniuse.com/mdn-css_properties_opacity_percentages), while Qt 5.12 is based on 69)

Reply:
Thank you for the review!

`display: none` breaks the `f` hinting :(
![breaks_f_hint](https://user-images.githubusercontent.com/36052282/144039482-5ca524ef-be8d-4bb3-845e-b5492ce05d00.png)

But `-webkit-appearance: none;` seems to work with the `f` hinting, yay! 
Also for the older version, as it is partially supported for [Chrome 4-81](https://caniuse.com/?search=-webkit-appearance)

I tried to write a few more beautiful comments, I hope that will help!